### ISSUE TITLE
create individual wiki

### ISSUE THREAD

**Issue by elmokadem** [Open-Systems-Pharmacology/Forum#133]
Hi,

Just a trivial point but below is a screenshot of the wiki page describing how PKSim creates an individual and I thought that the part in the red box is strangely placed. Should this part go under `STEP 2` section of the document where you discuss parameter scaling?
Perhaps I am just misinterpreting this.
![wikiscreenshot](https://user-images.githubusercontent.com/13800900/35694986-1d04002e-0751-11e8-9bb0-f7fef23b2ad1.png)

**Comment by msevestre** [comment]
@elmokadem *Scaled* is maybe not the right word. What is meant is that the volume of skin is not only randomly chosen from the distribution. Instead we also introduce a correlation with the body weight.

This needs to happen BEFORE the probability is calculated so location if fine I believe. Wording can be improved for sure. Please do not hesitate to submit your changes

**Comment by elmokadem** [comment]
Oh I see. That clarifies things. If I may say the wiki page is really helpful in clarifying a lot of the questions I had.

**Comment by msevestre** [comment]
@elmokadem Glad to hear. I have updated the section and removed the use of "scaled" to avoid any confusion in the future. Thanks

**Comment by elmokadem** [comment]
Hi again Michael,
So I am having a problem with an expression in the pseudo code for generating random volumes for lognormally distributed organs. This is the pseudo code provided:
```Return exp(Mean + Deviation*perturbation)```

Is this right? It seems that there is something wrong here. Could it be:
```Return exp(log(Mean) + log(Deviation)*perturbation)```

**Comment by msevestre** [comment]
@elmokadem 
Yep you are correct

This should read
 Mean * exp(log(Deviation) * perturbation);

See code [here](https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/6d50be080e18b310b6e4725c95aeeef8adee7367/src/PKSim.Core/Services/DistributedMuSigma.cs#L120)

**Comment by msevestre** [comment]
@elmokadem BTW: May I ask what you are trying to achieve? If you want to create individual from a scripting environment, (say Matlab) you can use some built in functions that will generate Individual and Populations using the PK-Sim implementation.

**Comment by elmokadem** [comment]
Thanks Michael. Indeed I want to create individuals and populations from R. Are you referring to the PKSim Matlab and R toolboxes when you say "built in functions"? Those have been buggy and I found trouble using them. Also, my understanding is that they need a PKSim output and I have found trouble implementing PKSim in a Mac environment where I am working.

**Comment by msevestre** [comment]
@elmokadem If you find any bug in the Toolboxes, I would suggest to post an issue in the dedicated repositories (either R ToolBox or Matlab ToolBox). Those Toolboxes are used extensively by many modelling teams so I am fairly confident that major bugs have been ironed out for a while. 

Of course, Mac OS is not supported so I am not surprised to hear that you are having issues. My suggestion would be to install a windows VM on your mac if you want to use the Toolboxes

**Comment by elmokadem** [comment]
Yes, I did that but for practical reasons, it is not that convenient to use the VM. That's why I am trying to recreate the mechanisms implemented in PKSim in R so that myself and any member of my company (Metrum Research Group), where we use mainly Macs, would be able to use the algorithms conveniently. If you have any ideas on how to do that in a more seamless manner I would really appreciate it.

**Comment by msevestre** [comment]
@elmokadem For obvious reasons, I cannot encourage you nor would recommend to duplicate the existing behavior of PK-Sim. [Parallels for Mac](https://www.parallels.com/ca/products/desktop/?gclid=EAIaIQobChMIpuGyorSZ2QIVAWSGCh0xVQT9EAAYASAAEgJZ2PD_BwE) provides a seemless integration between windows and mac. So much so in fact, that you do not even realize that you are running two different systems

That being said, we will investigate this year the possibility to move all PK-Sim core components to .NET Standard. Without being too technical, this should allow anyone to run PK-Sim CLI (command line interface) from Linux, Mac or Windows. The UI component will remain Windows only

**Comment by elmokadem** [comment]
I think that running the CLI from Mac would be a very good solution. In the meanwhile, I am not looking to replicate all of PKSim functionality, but just a compact, flow-limited version of a model that I already built using ICRP data. This seems to work quite well and I was able to regenerate the PKSim parameters to high accuracy. 
Still I will give Parallels for Mac a try as I haven't use it before.
